index.android.js

```diff
@@ -1,54 +1,5 @@
 import React, { Component } from 'react';
-import {
-  AppRegistry,
-  Image,
-  ListView,
-  StyleSheet,
-  Text,
-  View,
-  TouchableOpacity,
-} from 'react-native';
-import SplashScreen from './src/components/pages/SplashScreen';
-import LogOrSignPage from './src/components/pages/LogOrSignPage';
-import MapTest from './src/components/MapTest';
+import App from './src/index';
+import { AppRegistry } from 'react-native';
 
-export default class App2099 extends Component {
-  constructor() {
-    super();
-    this.state = {
-      loggedIn: false,
-    };
-  }
-    renderLoadingView() {
-      return (
-        <View style={styles.container}>
-          <Text>
-            Logged In...
-          </Text>
-        </View>
-      );
-    }
-
-  render() {
-    if (this.state.loggedIn){
-      renderLoadingView();
-    } else {
-      return (
-            <LogOrSignPage />
-
-      );
-    }
-  }
-}
-
-var styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    flexDirection: 'row',
-    justifyContent: 'center',
-    alignItems: 'center',
-    backgroundColor: '#F5FCFF',
-  },
-});
-
-AppRegistry.registerComponent('App2099', () => App2099);
+AppRegistry.registerComponent('App2099', () => App);
```

Comment:
what's this app name? App2099 => App, are they important at all?

Reply:
yes, we need to register the app, at the very beginning. 

App2099 on the right of the arrow made it seem specific to our app but instead App is more descriptive to just being the App app itself.

At the end its just a naming convention.